simple_rpc/simple_rpc.py

```diff
@@ -103,14 +133,7 @@ def _get_methods(self):
 
     def open(self):
```

Comment:
This function is only called once by the constructor if `autoconnect==True`.
Alternatively, it is called explicitly if `autoconnect==False` or when an explicit `close` was done.

Reply:
ok